README.md

```diff
@@ -13,154 +17,55 @@ To get started, there are some basic prerequisites which must be installed:
 * [Docker Compose](https://docs.docker.com/compose/install/)
 * [GNU Make v4.0+](https://www.gnu.org/software/make/)
 
-In addition, a working knowledge of GraphQL is assumed. If you are new to this,
-a good starting point is [Introduction to GraphQL](https://graphql.org/learn/).
-
-## Clone the Repository
-
-```bash
-git clone https://github.com/btpworks/chronicle-examples.git
-```
+## Setting up your own domain
 
-This contains several example domain yaml files, and Docker uses
-`blockchaintp/chronicle-builder:BTP2.1.0-0.6.0` as the builder image by default
+This repository follow the same structure as the [Chronicle Examples](https://github.com/btpworks/chronicle-examples)
+repository which you can use for reference.
 
-## Build a Domain
+1. Clone this repo, or download it as a zip file from GitHub
+   [here](https://github.com/btpworks/chronicle-bootstrap/archive/refs/heads/main.zip).
+   *Please note, if you download or copy the repo rather than cloning it,
+   you will need to make sure you keep it up to date with the latest Chronicle
+   releases in the future.*
+1. Add your domain.yaml file in its own directory inside the
+   `domains` directory. Both the name of the directory and the name of the yaml
+   file should be the same as the name of your domain.
+   For example, if your domain is called `manufacturing`, you would
+   create `domains/manufacturing/manufacturing.yaml`
```

Comment:
maybe link to the file in `chronicle-examples`?

Reply:
I've added a link